src/expr.rs

```diff
@@ -35,7 +37,222 @@ pub enum Expr {
     Geometry(Geometry),
 }
 
+impl From<Value> for Expr {
+    fn from(v: Value) -> Expr {
+        let e: Expr = serde_json::from_value(v).unwrap();
+        e
+    }
+}
+
+impl From<Expr> for Value {
+    fn from(v: Expr) -> Value {
+        let v: Value = serde_json::to_value(v).unwrap();
+        v
+    }
+}
+
+
+impl TryInto<f64> for Expr {
```

Comment:
You should implement `TryFrom` instead of `TryInto`: https://doc.rust-lang.org/std/convert/trait.TryInto.html